lib/ex_aws/config/defaults.ex

```diff
@@ -17,7 +17,7 @@ defmodule ExAws.Config.Defaults do
       region: "us-east-1",
       port: 80
     },
-    kinesis_firehose: %{
+    firehose: %{
```

Comment:
Although it's kind of verbose I'd prefer if we kept this as `kinesis_firehose` unless we're also going to change the module from `KinesisFirehose` to `Firehose`.